test/system/710-kube.bats

```diff
@@ -157,4 +157,44 @@ status  | =  | {}
     run_podman rmi $(pause_image)
 }
 
+# kube play --wait=true, where we clear up the created containers, pods, and volumes when a kill or sigterm is triggered
+@test "podman kube play --wait" {
+    cname=c$(random_string 15)
+    fname="/tmp/play_kube_wait_$(random_string 6).yaml"
+    run_podman container create --name $cname $IMAGE top
+    run_podman kube generate -f $fname $cname
+
+    # delete the container we generated form
```

Comment:
I think this might be a typo for `from`?

Reply:
fixed